runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -120,6 +138,20 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
     @Override
     public <R> R access(ICorfuSMRAccess<R, T> accessMethod,
                         Object[] conflictObject) {
+        R value;
+        boolean isEnabled = MetricsUtils.isMetricsCollectionEnabled();
```

Comment:
The one case so far of related metrics is the server's operation timer (which is a counter + histogram) and the `counterTokenSum` and `counterToken0` simple counters.  If the timer is updated for any given operation, then the simple counters should always be updated also.  If the timer isn't updated, then counters shouldn't be either.